architecture/src/androidLibMain/kotlin/navigation/Navigator.kt

```diff
@@ -156,8 +177,19 @@ class ActivityNavigator<A : NavigationAction<*>>(private val navigationMapper: (
         val settings = emailSpec.emailSettings
         val intent = when (settings.attachments.size) {
             0 -> Intent(Intent.ACTION_SEND)
-            1 -> Intent(Intent.ACTION_SEND).apply { putExtra(Intent.EXTRA_STREAM, settings.attachments[0]) }
-            else -> Intent(Intent.ACTION_SEND_MULTIPLE).apply { putExtra(Intent.EXTRA_STREAM, ArrayList(settings.attachments)) }
+            1 -> Intent(Intent.ACTION_SEND).apply {
```

Comment:
Noticed that this part looks duplicated when looking at the same `when` block in other functions, for example navigateToMEssenger in this PR.
You could probably group this in a function.. I'd use a name like `bindAttachmentToSendIntent` or `generateSendActionIntent`